ColorCodeMngmnt_Verification.c

```diff
@@ -0,0 +1,91 @@
+/******************************************************************************
+*******     (C) COPYRIGHT Robert Bosch GmbH  - All Rights Reserved     ********
+******************************************************************************/
+/*!
+***     \file        ColorCodeMngmnt.c
+***     \author      Abinesh KanjiKovil Thatchinamoorthy
+***     \brief       Source Code of verifying Color Code Management for cables
+***                  
+*****************************************************************************/
+
+/* Project Inlusions */
+#include "ColorCodeMngmnt.h"
+
+/*=============================================================================
+=======                      FUNCTION DECLARATIONS                      =======
+=============================================================================*/
+static void testNumberToPair_v(int pairNumber_i,
+                        majorColor_t expectedMajor_e,
+                        minorColor_t expectedMinor_e);
+static void testPairToNumber_v(majorColor_t majorColor_e,
+                        minorColor_t minorColor_e,
+                        int expectedPairNumber_i);
+
+/*=============================================================================
+=======                       FUNCTION DEFINITIONS                      =======
+=============================================================================*/
+
+/*---------------------------------------------------------------------------*/
+/*     FUNCTION:    testNumberToPair_v
+ */
+/*!    \brief       Tests whether the input color names are the corresponding 
+ *                  names of the provided pair number
+ *
+ *     \param       pairNumber_i [IN] - Pairnumber for which the verification
+                    has to be done (int)
+ *     \param       expectedMajor_e [IN] -  Expected major Color for the
+                    provided pair number(minorColor_t)
+ *     \param       expectedMinor_e [IN] - Expected minor Color for the
+                    provided pair number(majorColor_t)
+*     \returns     void
+*//*------------------------------------------------------------------------*/
+static void testNumberToPair_v(int pairNumber_i,
+                        majorColor_t expectedMajor_e,
```

Comment:
Try to think of a test-name that includes the expectation of the test. Do you get more test possibilities?